src/platform/graphics/graphics-device.js

```diff
@@ -444,7 +444,7 @@ class GraphicsDevice extends EventHandler {
     }
 
     updateClientRect() {
-        this.clientRect = this.canvas.getBoundingClientRect();
+        this.clientRect = new DOMRect(0, 0, this.canvas.width, this.canvas.height);
```

Comment:
Couple of issues here:

1. `DOMRect` isn't available in Node.js so this causes the unit tests to fail.
2. This function is called every frame so this will be generating garbage. Probably not a significant amount - just pointing it out.